src/Pages/CreateProduct/components/ProductForm/index.tsx

```diff
@@ -6,11 +6,17 @@ import { Button } from "primereact/button";
 import { useMutation } from "react-query";
 import { URL } from "../../../../constants";
 import { ProductService } from "../../../../services/product-service";
-import { useNavigate } from "react-router-dom";
 import { InputField } from "../../../../components/UI/Forms/InputField";
 import { validationYup } from "../../../../schema/validations/validationSchema";
 import { useTranslation } from "react-i18next";
 
+interface IProps {
+  productData?: IProduct;
+  handleSuccess: () => void;
+  onCancel?: () => void;
```

Comment:
```suggestion
  onCancel: () => void;
```

Reply:
I pass it only in the Edit component,in the CreateProduct component i dont need to pass it(onCancel)